be/src/exprs/anyval_util.cpp

```diff
@@ -164,10 +166,64 @@ FunctionContext::TypeDesc AnyValUtil::column_type_to_type_desc(const TypeDescrip
     case TYPE_NULL:
         out.type = FunctionContext::TYPE_NULL;
         break;
+    case TYPE_ARRAY: {
+        out.type = FunctionContext::TYPE_ARRAY;
+        for (auto&& t : type.children) {
```

Comment:
why use right value reference here?

Reply:
Em，the better way is `const auto &`, will modify it.